portal/static/js/main.js

```diff
@@ -2592,7 +2592,53 @@ var Profile = function(subjectId, currentUserId) {
     };
     this.initPatientEmailFormSection = function() {
         var self = this;
-        if (!hasValue($("#email").val())) $("#profileEmailSelect").attr("disabled", true);
+        if (!$("#email").val()) {
+            $("#profileEmailSelect").attr("disabled", true);
+        };
+
+        if ($("#assessmentStatusContainer").length > 0) {
+            tnthAjax.patientReport(self.subjectId, function(data) {
+                if (!data.error) {
+                    var pcpReports = data["user_documents"]? $.grep(data["user_documents"], function(document) {
+                        return /P3P/gi.test(document.filename);
+                    }): false;
+                    var hasReports = pcpReports && pcpReports.length > 0;
+                    /*
+                     * adjust select/send email UI only if email is available
+                     */
+                    if (hasValue($("#email").val())) {
+                        if (!hasReports) {
```

Comment:
It might make sense to have a single if/else block on `hasReports`, then do these specific checks (eg `hasValue($("#email").val())`) inside that block, rather than checking `hasReports` multiple times.